<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddItemForm</class>
 <widget class="QWidget" name="AddItemForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление услуги/товара</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Наименование товара/услуги:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Номер заказа:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="orderID_lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>351</width>
     <height>22</height>
    </rect>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="itemName_lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>351</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Количество:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="quantity_spinBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Стоимость:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cost_lineEdit">
   <property name="geometry">
    <rect>
     <x>282</x>
     <y>140</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addItem_pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
